pkg/configs/db/postgres/postgres.go

```diff
@@ -225,7 +226,7 @@ func (d DB) findRulesConfigs(filter squirrel.Sqlizer) (map[string]configs.Versio
 		var cfg configs.VersionedRulesConfig
 		var userID string
 		var cfgBytes []byte
-		err = rows.Scan(&cfg.ID, &userID, &cfgBytes)
+		err = rows.Scan(&cfg.ID, &userID, &cfgBytes, &cfg.DeletedAt)
```

Comment:
This `cfg.DeletedAt` has to be time or null to be able to store null time from DB.